### DISCUSSION TITLE
🔨 iLogtail开发者双周会（10月19号 19:30-20:30）

### DISCUSSION THREAD

**Discussion by yyuuttaaoo** [alibaba/ilogtail#453] [Announcements]
# iLogtail双周会
时间：10月19日 周三 19:30 - 20:30
会议号：474 394 18279
电话呼入：+862281944287（中国大陆）+862281944280（中国大陆）
入会链接：dingtalk://dingtalkclient/page/videoConfFromCalendar?confId=c5a294b0-d725-41c7-9a18-0cf9b92bce35&appendCalendarId=1&calendarId=2017305868

## 议题

* iLogtail社区Roadmap进展（Release、Contributor、Conference） @yyuuttaaoo
* Kafka Flusher进展 @shalousun
* OTLP输入、输出支持进展和后续计划 @liuhaoyang
* iLogtail与其他开源Agent性能比较 @XLPE
* [HTTP Flusher需求讨论](https://github.com/alibaba/ilogtail/issues/450) @totticarter
* 代码库去除vendor依赖和Agent管控协议标准讨论 @Takuka0311
* 社区版近期重点工作计划讨论

会议同时欢迎，任何使用经验、吐槽、端采集技术分享。

**Comment by messixukejia** [comment]
@EvanLjp 增加一个ebpf开发规范的议题吧

**Reply by EvanLjp** [reply]
下次分享，本次时间来不及

**Comment by messixukejia** [comment]
@yyuuttaaoo 增加下这个议题https://mp.weixin.qq.com/s/94wiV7YhaeXfria6x21JeA

**Comment by yyuuttaaoo** [comment]
后续重点工作
1. metric、trace模型
2. configserver
3. ck flusher
4. pulsar flusher
5. bugfix （offset过了插件不生效 #395，容器采集重复 #420，input service_kafka报错 #471）
6. self monitoring #75

**Comment by yyuuttaaoo** [comment]
双周会回放视频 https://www.bilibili.com/video/BV128411e7XF/?spm_id_from=333.999.0.0
